definitions/ext-network_dhcp_service/definition.yml

```diff
@@ -0,0 +1,38 @@
+domain: EXT
+type: NETWORK_DHCP_SERVICE
+synthesis:
+  rules:
+  # IPv4 DHCP Service via Gigamon
+  - identifier: dst_ip
+    name: dst_ip
+    conditions:
+    - attribute: app_name
+      value: dhcp
+    - attribute: instrumentation.provider
+      value: Gigamon
+    - attribute: dhcp_ip_lease_time
+      present: true
+    - attribute: dst_ip
+      present: true
```

Comment:
nit: this is implicit with `identifier: dst_ip` so you can remove it.

Reply:
Done